svgs/general/code.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><path d="M0 0h24v24H0V0z"/><path d="M9.4 16.6L4.8 12l4.6-4.6L8 6l-6 6 6 6 1.4-1.4zm5.2 0l4.6-4.6-4.6-4.6L16 6l6 6-6 6-1.4-1.4z"/></svg>
```

Comment:
this is showing up as a black box.
<img width="740" alt="Screen Shot 2019-06-12 at 9 43 12 AM" src="https://user-images.githubusercontent.com/306275/59370053-93bd2200-8cf6-11e9-8420-f191183147c7.png">

when you click the paper icon:
<img width="263" alt="Screen Shot 2019-06-12 at 9 43 57 AM" src="https://user-images.githubusercontent.com/306275/59370086-a5062e80-8cf6-11e9-9ae0-77791cc3e314.png">


if you remove `<path d="M0 0h24v24H0V0z"/>`, it is fixed. it was the path with `fill="none"` before and doesn't look like it is part of the main svg path.

Reply:
Thanks! I've corrected and verified in local storybook, it should now work per your suggestion.

Reply:
hrmm did you push the changes? still seeing that unused path in the source code and generated component

update the base svg from:
```svg
<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24">
  <path fill="none" d="M0 0h24v24H0V0z"/>
  <path d="M9.4 16.6L4.8 12l4.6-4.6L8 6l-6 6 6 6 1.4-1.4zm5.2 0l4.6-4.6-4.6-4.6L16 6l6 6-6 6-1.4-1.4z"/>
</svg>
```

to:
```svg
<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24">
  <path d="M9.4 16.6L4.8 12l4.6-4.6L8 6l-6 6 6 6 1.4-1.4zm5.2 0l4.6-4.6-4.6-4.6L16 6l6 6-6 6-1.4-1.4z"/>
</svg>
```